public/app/features/alerting/unified/components/rule-viewer/v2/RuleViewer.v2.tsx

```diff
@@ -184,53 +207,28 @@ interface TitleProps {
   name: string;
   // recording rules don't have a state
   state?: PromAlertingRuleState;
+  health?: RuleHealth;
+  ruleType?: PromRuleType;
 }
 
-export const Title = ({ name, state }: TitleProps) => (
-  <div style={{ display: 'flex', alignItems: 'center', gap: 8, maxWidth: '100%' }}>
-    <LinkButton variant="secondary" icon="angle-left" href="/alerting/list" />
-    <Text element="h1" truncate>
-      {name}
-    </Text>
-    {/* recording rules won't have a state */}
-    {state && <StateBadge state={state} />}
-  </div>
-);
-
-interface StateBadgeProps {
-  state: PromAlertingRuleState;
-}
-
-// TODO move to separate component
-const StateBadge = ({ state }: StateBadgeProps) => {
-  let stateLabel: string;
-  let textColor: 'success' | 'error' | 'warning';
-
-  switch (state) {
-    case PromAlertingRuleState.Inactive:
-      textColor = 'success';
-      stateLabel = 'Normal';
-      break;
-    case PromAlertingRuleState.Firing:
-      textColor = 'error';
-      stateLabel = 'Firing';
-      break;
-    case PromAlertingRuleState.Pending:
-      textColor = 'warning';
-      stateLabel = 'Pending';
-      break;
-  }
+export const Title = ({ name, state, health, ruleType }: TitleProps) => {
+  const isRecordingRule = ruleType === PromRuleType.Recording;
 
   return (
-    <Stack direction="row" gap={0.5}>
-      <AlertStateDot size="md" state={state} />
-      <Text variant="bodySmall" color={textColor}>
-        {stateLabel}
+    <div style={{ display: 'flex', alignItems: 'center', gap: 8, maxWidth: '100%' }}>
```

Comment:
do you use inline css un purpose in here?

Reply:
Oops no, I don't think so – there's certainly no reason for it so I'll convert this to a proper `className` :D